Makefile

```diff
@@ -63,7 +63,8 @@ release-major release-minor release-patch:
 
 .PHONY: setup
 setup:
-	$(NPM) install
+	$(YARN)
+	$(YARN) check
```

Comment:
If I use the dot, I get this output:

`make: Nothing to be done for 'setup'.`

Replacing hte dot with an underscore makes the command run properly.